Error building cli/encore from source in windows 10.

1. When I am building encore cli in windows, getting compilation error `..\..\daemon\dash\server.go:25:12: pattern dashapp/dist/*: no matching files found`

<img width="708" alt="encore-error" src="https://user-images.githubusercontent.com/22986964/129436515-2882e6aa-3c6c-44f4-bebd-e56d662b7149.PNG">


So to compile this, I have removed this line https://github.com/encoredev/encore/blob/67aefe20466f686561c300dd1b8d6276a563b2bc/cli/daemon/dash/server.go#L25

2. After it compiles, then when I run `./encore.exe -h` it panics.
<img width="669" alt="encore-panic" src="https://user-images.githubusercontent.com/22986964/129437330-9872ad19-9726-40fd-bb3f-a73171847c10.PNG">

This seems issue in wireguard/windows dependency. So after I update wireguard from v0.3.1 to v0.4, it works. But I feel like this can have impact somewhere else as well.
<img width="537" alt="encore-mod" src="https://user-images.githubusercontent.com/22986964/129437460-364b34fa-6e1a-4bab-b97c-e74db876a4e1.PNG">

Please let me know what will be the permanent solution for both the issues, and if upgrading wireguard is only solution then I will pull request for it.

Thanks.

--- a/cli/internal/wgtunnel/wgtunnel_darwin.go
+++ b/cli/internal/wgtunnel/wgtunnel_darwin.go
@@ -16,6 +16,7 @@ import (
 	"syscall"
 	"time"
 
+	"golang.zx2c4.com/wireguard/conn"
 	"golang.zx2c4.com/wireguard/device"
 	"golang.zx2c4.com/wireguard/ipc"
 	"golang.zx2c4.com/wireguard/tun"
@@ -279,7 +280,7 @@ func run() error {
 		"vpn: ",
 	)
 
-	device := device.NewDevice(tun, logger)
+	device := device.NewDevice(tun, conn.NewDefaultBind(), logger)
 
 	term := make(chan os.Signal, 1)
 	errs := make(chan error)

--- a/cli/internal/wgtunnel/wgtunnel_linux.go
+++ b/cli/internal/wgtunnel/wgtunnel_linux.go
@@ -15,6 +15,7 @@ import (
 	"syscall"
 	"time"
 
+	"golang.zx2c4.com/wireguard/conn"
 	"golang.zx2c4.com/wireguard/device"
 	"golang.zx2c4.com/wireguard/ipc"
 	"golang.zx2c4.com/wireguard/tun"
@@ -259,7 +260,7 @@ func run() error {
 		"vpn: ",
 	)
 
-	device := device.NewDevice(tun, logger)
+	device := device.NewDevice(tun, conn.NewDefaultBind(), logger)
 
 	term := make(chan os.Signal, 1)
 	errs := make(chan error)

--- a/go.mod
+++ b/go.mod
@@ -31,7 +31,7 @@ require (
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
 	golang.org/x/sys v0.0.0-20210630005230-0f9fa26af87c
 	golang.org/x/tools v0.0.0-20200825202427-b303f430e36d
-	golang.zx2c4.com/wireguard v0.0.20201118
+	golang.zx2c4.com/wireguard v0.0.0-20210728232740-bad6caeb82ed
 	golang.zx2c4.com/wireguard/wgctrl v0.0.0-20210803171230-4253848d036c
 	golang.zx2c4.com/wireguard/windows v0.4.1
 	google.golang.org/genproto v0.0.0-20201030142918-24207fddd1c3

